scripts/tests/test-ci-all.sh

```diff
@@ -5,6 +5,9 @@ set -euo pipefail
 # prevent locale settings messing with some setups
 export LANG=C
 
+# run with 1 guardian offline
+export FM_DEVIMINT_DEGRADED=1
```

Comment:
This just sets a default to run all these tests with 1 guardians offline. I'm not sure what's best to do here.